### ISSUE TITLE
Support the inverse annotation

### ISSUE THREAD

**Issue by OpenModelica-TracImporter** [OpenModelica/OpenModelica#5300]

**Comment by thorade** [comment]
https://trac.openmodelica.org/OpenModelica/ticket/5300
https://github.com/modelica/ModelicaSpecification/issues/2942

**Comment by thorade** [comment]
Is this still open?
#7416 sounds like it is done already

**Comment by perost** [comment]
Yes, #7416 was just about passing the inverse annotations through the frontend (and e.g. instantiating the inverse functions) to enable our backend developers to use them. But I don't think there's been anything done in the backend yet, so they're currently just passed on and then ignored.

**Comment by casella** [comment]
We are implementing a compiler at Politecnico which relies on the OMC frontend, so we needed it to provide this information, hence #7567. It is currently not used in the regular OMC backend. We could prioritize it if there are good reasons/applications for it.

**Comment by thorade** [comment]
I came here after reading/commenting https://github.com/modelica/ModelicaSpecification/issues/2942 
No need to re-prioritize any OMC work.

**Comment by YomGarcia** [comment]
Any news regarding inverse support in the regular OMC backend? We could see a use case for implemented external inverse functions

**Comment by casella** [comment]
@YomGarcia at the moment we are prioritizing the development of the new backend, so there's not much work going on in the old one, except maintenance.

@kabdelhak, @phannebohm can you comment about the support of inverse annotations in the NB?

**Comment by phannebohm** [comment]
I believe inverse functions are not yet used for anything in the NB.

As soon as we have symbolic solving beyond linear equations, we'll deal with the inverse annotation.
